src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+// A-Enums: Set Enums for Task type
+enum TASK_TYPE {
+    T,D,E
+}
+
 public class Duke {
+
+    // shows if the Duke chat bot has been activated
+    private boolean activated;
```

Comment:
Should this be named to sound more like a boolean?

```suggestion
    private boolean hasActivated;
```